docs/setting-up.md

```diff
@@ -50,14 +50,24 @@ More information can be found at [this documentation](https://help.github.com/ar
 
    # Linux/OS X
    ./install.sh --path-update true
+
    # Windows
    install.bat --path-update true
    ```
+   If you are installing in Red Hat/CentOS/Fedora/Debian/Ubuntu follow the official documentation of Google Cloud Engine for [Debian/Ubuntu](https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu) or [RHEL/CentOS/Fedora](https://cloud.google.com/sdk/docs/quickstart-redhat-centos) respectively.
```

Comment:
Minor readability tweaks:
- Red Hat/CentOS/Fedora/Debian/Ubuntu -> Red Hat, CentOS, Fedora, Debian, or Ubuntu
- Ubuntu follow -> Ubuntu, refer to
- official documentation -> quickstart guide
- Google Cloud Engine -> Google Cloud SDK
- RHEL -> Red Hat

Reply:
On it... :wink: